src/ResourceManager/Compute/Commands.Compute/Extension/AEM/SetAzureRmVMAEMExtension.cs

```diff
@@ -61,11 +61,18 @@ public class SetAzureRmVMAEMExtension : VirtualMachineExtensionBaseCmdlet
                 Mandatory = false,
                 Position = 2,
                 ValueFromPipelineByPropertyName = false,
-                HelpMessage = "If this parameter is provided, the commandlet will not enable Windows Azure Diagnostics for this virtual machine.")]
+                HelpMessage = "Deprecated - Windows Azure Diagnostics is now disabled by default")]
         public SwitchParameter DisableWAD { get; set; }
 
         [Parameter(
                 Mandatory = false,
+                Position = 2,
```

Comment:
@MSSedusch switch parameters do not need to have a position. You can remove this from both `EnableWAD` and `DisableWAD`